config/electronBuilder/electronBuilder.ghactions.config.js

```diff
@@ -0,0 +1,13 @@
+const baseElectronBuilderConfig = require('./electronBuilder.local.config');
+
+const isStagingBuild = process.env.NODE_ENV === 'staging';
+
+module.exports = {
+    ...baseElectronBuilderConfig,
+    publish: [{
+        provider: 's3',
+        bucket: isStagingBuild ? 'staging-expensify-cash' : 'expensify-cash',
```

Comment:
```suggestion
        bucket: process.env.NODE_ENV === 'staging' ? 'staging-expensify-cash' : 'expensify-cash',
```
I think this is fine, and the extra variable really isn't necessary.